scanner/updater/export.go

```diff
@@ -217,3 +219,26 @@ func (t *rateLimitedTransport) RoundTrip(req *http.Request) (*http.Response, err
 	}
 	return t.transport.RoundTrip(req)
 }
+
+func epssOpts(ctx context.Context) []updates.ManagerOption {
+	enricher := &epss.Enricher{}
+	configJSON, err := json.Marshal(epss.Config{
+		URL: nil, // use default URL for now
```

Comment:
I did it this way because I think we might want to cache the EPSS data since it only gets updated once per day.